src/views/PaymentPageView.vue

```diff
@@ -26,24 +28,68 @@ export default {
   computed: {
     ...mapStores(usePaymentStore),
     paymentQrCodeUrl: function (): String {
-      return this.paymentsStore.invoice.payment_request?.qr_code_url || '' // todo: show an image to make it obvious this failed
+      return this.paymentsStore.invoice.payment_request?.qr_code_url || '' // TODO: show an image to make it obvious this failed
     },
     awaitingPayment: function (): boolean {
+      if (this.missingParams) {
```

Comment:
Maybe rather navigate to error if something is wrong: i.e. using https://router.vuejs.org/guide/essentials/navigation.html